app/src/main/java/org/sopt/official/feature/attendance/compose/component/AttendanceCodeCardList.kt

```diff
@@ -0,0 +1,42 @@
+package org.sopt.official.feature.attendance.compose.component
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.official.designsystem.SoptTheme
+
+@Composable
+fun AttendanceCodeCardList(
+    codes: List<String?>,
+    onTextChange: (newText: String) -> Unit,
+    onTextFieldFull: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Row(modifier = modifier) {
+        repeat(codes.size) { index ->
+            AttendanceCodeCard(
+                text = codes[index] ?: "",
+                onTextChange = onTextChange,
+                onTextFieldFull = onTextFieldFull
+            )
+            if (index < codes.size) {
+                Spacer(modifier = Modifier.width(width = 12.dp))
+            }
+        }
```

Comment:
```suggestion
        codes.forEachIndexed { item, index ->
            AttendanceCodeCard(
                text = item ?: "",
                onTextChange = onTextChange,
                onTextFieldFull = onTextFieldFull
            )
            if (index < codes.size) {
                Spacer(modifier = Modifier.width(width = 12.dp))
            }
        }
```

(개인취향) forEachIndexed로 써도 될 것 같음